ffv1.md

```diff
@@ -71,7 +71,7 @@ Note: the operators and the order of precedence are the same as used in the C pr
 
 `a | b`       means bit-wise "or" of a and b.
 
-`a >> b`      means arithmetic right shift of two’s complement integer representation of a by b binary digits.
+`a >> b`      means arithmetic right shift of two’s complement integer representation of a by b binary digits. This is equivalent to, b times dividing a by 2 with rounding toward negative infinity.
```

Comment:
Rewording suggestion:
`This is equivalent to the largest integer less than or equal to a divided by 2 for b times.`

Reply:
what definition of divided does this use. its not what C uses as that rounds to 0 and you no longer  change the rounding so i think this is ambigous.

Reply:
I agree that rounding toward negative infinity is important to state.